src/components/CarouselItem.js

```diff
@@ -35,11 +37,11 @@ class CarouselItem extends PureComponent {
           },
         )}
         style={{
-          paddingRight: `${this.props.offset / 2}px`,
-          paddingLeft: `${this.props.offset / 2}px`,
-          width: `${this.props.width}px`,
-          maxWidth: `${this.props.width}px`,
-          minWidth: `${this.props.width}px`,
+          paddingRight: `${this.props.offset / 2}${this.props.offsetUnit}`,
```

Comment:
I'm not sure how it works. If everything is calculated in pixels and you will switch just the unit that will multiply everything. I think you should calculate everything in rems. Did you test that solution?

Reply:
I checked sometimes `%` works even better than `px`, `em` or `rem`
https://recordit.co/5dpI9Adpcv